web3.js/test/transaction.test.js

```diff
@@ -92,7 +92,20 @@ test('partialSign', () => {
   partialTransaction.setSigners(account1.publicKey, account2.publicKey);
   expect(partialTransaction.signatures[0].signature).toBeNull();
   expect(partialTransaction.signatures[1].signature).toBeNull();
-  partialTransaction.partialSign(account1, account2);
+
+  partialTransaction.partialSign(account1);
+  expect(partialTransaction.signatures[0].signature).not.toBeNull();
+  expect(partialTransaction.signatures[1].signature).toBeNull();
+
+  expect(() => partialTransaction.serialize()).toThrow();
+  expect(() => partialTransaction.serialize(false)).not.toThrow();
```

Comment:
I think we should have a test that verifies that the transaction is reconstructed properly with `Transaction.from` when signatures are `null`

Reply:
Yeah there's some slightly problems here.  The `isSigner` field doesn't seem to get restored as expected.  Good idea but I'm going to punt on adding test to this PR though.